cmd/cockroach-operator/BUILD.bazel

```diff
@@ -38,7 +38,7 @@ container_run_and_commit_layer(
     name = "ubi_update",
     commands = [
         "microdnf install yum",
-        "yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical",
```

Comment:
Do we need the -v?

Reply:
Not really, but it may help debugging in the future. 🤷

Reply:
Probably not but it doesn't hurt, just gives the verbose output